### PR TITLE
Bump ReactiveCocoa to Release Candidate 1 and add Rex

### PR DESCRIPTION
**Pull request by bjarkehs**
A minor bump to ReactiveCocoa now using their versioning for the pod. Also added a podspec for Rex, since it's a nice addition to ReactiveCocoa.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Cool! The changes to the RAC podspec look great – not sure about Rex, I'll add some inline comments.

File: Rex.podspec [hunk 1]
```diff
@@ -0,0 +1,136 @@
+#
```
Comment (ashfurrow) [inline Rex.podspec:1]: Generally, we should strip out comments like this from Podspecs. Anything that starts with `#` basically

File: Rex.podspec [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  # s.platform     = :ios
+  # s.platform     = :ios, "8.0"
+
+  #  When using multiple platforms
+  s.ios.deployment_target = "8.0"
+  s.osx.deployment_target = "10.10"
+  # s.watchos.deployment_target = "2.0"
```
Comment (ashfurrow) [inline Rex.podspec:69]: Are there any valid deployment targets for Rex? It'd be cool to include those, too 🎉

**Comment by ashfurrow** [comment]
Actually, if the Rex library is separate from ReactiveCocoa, it might be worth submitting a PR to https://github.com/neilpa/Rex with the podspec instead. @neilpa may be interested in including the podspec as part of their repo so the most developers can use it 🎉

**Comment by bjarkehs** [comment]
Yeah Christmas season kinda happened. So I'm going to instead make a new Rex podspec and open a pull request at https://github.com/neilpa/Rex. I appreciate the feedback though, thanks a lot :tada:

**Pull request closed without merging**